app/helpers/timed_update_helper.rb

```diff
@@ -0,0 +1,5 @@
+module TimedUpdateHelper
+  def before_update_time?(year:, month:, day:, hour:, minute:)
```

Comment:
Minor: I'm not sure about this name, but I can't think of anything better!

Reply:
Yeah, I also couldn't think of anything better. Let's sleep on it and see if a better name turns up in the morning.